packages/protocol/src/interaction/InteractionServer.ts

```diff
@@ -540,14 +541,15 @@ export class InteractionServer implements ProtocolHandler, InteractionRecipient
                         attributeData: { path, dataVersion: version, payload: value, schema },
                     };
                 } catch (error) {
-                    logger.error(
-                        `Error while reading attribute from ${
-                            exchange.channel.name
-                        } to ${this.#endpointStructure.resolveAttributeName(path)}:`,
-                        error,
-                    );
+                    const what = `reading ${this.#endpointStructure.resolveAttributeName(path)} from ${exchange.channel.name}`;
+
+                    if (!(error instanceof StatusResponseError)) {
+                        const wrappedError = new ImplementationError(`Unhandled error ${what}`);
```

Comment:
Would InternalError be better? ImplementationError was more thought like "hey dear developer, check your code". But when read fails it is more like "our side"?

Reply:
Yes maybe so... I used ImplementationError because there could in theory be errors from e.g. dynamic attributes, but does seem more likely is our fault.

Maybe we should add a new UnhandledError or something we can use in cases where we don't know whose fault the error is?

Reply:
Actually though I should trigger wtf conditionally if it's enabled via command line flags

Reply:
Replied to wrong thread there 🙄

Reply:
Ok, lets leave ImplementationError